in-vehicle-digital-twin/src/provider_impl.rs

```diff
@@ -37,35 +35,29 @@ impl Provider for ProviderImpl {
         &self,
         request: Request<UnsubscribeRequest>,
     ) -> Result<Response<UnsubscribeResponse>, Status> {
-        info!("Got an unsubscribe request: {:?}", request);
-        // TODO - provide unsubscribe functionality
-        let response = UnsubscribeResponse {};
+        warn!("Got an unsubscribe request: {request:?}");
 
-        Ok(Response::new(response))
+        Err(Status::unimplemented("unsubscribe has not been implemented"))
```

Comment:
here and at other places - consider having '{classname::methodname} has not been implemented'. Or have something like this - https://stackoverflow.com/a/63904992/2147244